Felienne/hedy
# When subscribed user changes email, remove old & add new email to Mailchimp list

- Closes #816 
- If a user subscribed to the mailing list changes email address, unsubscribe the old email address and subscribe the new one using the Mailchimp API.
- Implementation notes: the subscription logic, now called from two places, is now inside a helper function.

**How to test**:
- Change the email address of a user that is subscribed to the newsletter and verify in Mailchimp that the old contact is archived and the new one is added to the list.
